.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: ci
+on:
+  push:
+    branches:
+    - master
```

Comment:
Should this only be run on `master`? Wouldn't we want this to run on push to any branch?

Reply:
Hmm, well it all ran on _this_ branch... admittedly I cribbed this from frontend-app-account.

Reply:
Ah, it's the `pull_request` below that gives us what we want.  That causes it to run on PRs.

Reply:
OH, sorry. The `pull_request` was already in there?! I missed that in the initial review, my bad!